<launch>
    <node name="joy_node" pkg="joy" type="joy_node">
    <param name="autorepeat_rate" value="20" />
    <param name="deadzone" value="0.10" />
    </node>

    <param name="alpha" value="0.30"/>
    <param name="beta" value="0.31"/>
    <param name="wheel_radius" value="0.075"/>
    <param name="max_linear_vel" value="1.0"/>
    <!-- parameter "angular_vel_div" is the divisor of pi to compute max angular velocity -->
    <!-- for example, if angular_vel_div=2, then max angular velocity is pi/2 -->
    <param name="angular_vel_div" value="2"/>

    <node name="wm_mecanum_teleop_node" pkg="wm_mecanum_cmd" type="wm_mecanum_teleop_node.py"/> 

    <node name="wm_mecanum_cmd_node" pkg="wm_mecanum_cmd" type="wm_mecanum_cmd_node.py"/>

</launch>

